### PR TITLE
[DO-580] add links to SDK supplementaries

### PR DESCRIPTION
**Pull request by Hdd87**

### CODE REVIEW

File: content/2.application.md [hunk 1]
```diff
@@ -55,3 +55,12 @@ To simplify the development of a custom Payments CorDapp in the absence of the G
...
 The PSP payments gateway is an endpoint of a proprietary off-chain payments solution on the PSP side.
+
+### Supplementary parts of the Payments SDK
+
+1. [ISO20022 XML](https://github.com/gevamu/corda-payments-sdk/tree/master/iso20022)
+2. Payments App Sample:
+    - [Payments App Backend - Gevamu](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-app-sample/payments-app-backend)
+    - [Payments App Frontend - Client App](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-app-sample/payments-app-frontend)
+3. [Payments-Contracts](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-contracts)
+4. [Payments-Workflows](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-workflows)
```
Comment (d0rich) [inline content/2.application.md:66]: These are 2 parts of SDK. I would group these points and place it on the first place.

File: content/2.application.md [hunk 2]
```diff
@@ -55,3 +55,12 @@ To simplify the development of a custom Payments CorDapp in the absence of the G
 
 ### PSP payments gateway
 The PSP payments gateway is an endpoint of a proprietary off-chain payments solution on the PSP side.
+
+### Supplementary parts of the Payments SDK
+
+1. [ISO20022 XML](https://github.com/gevamu/corda-payments-sdk/tree/master/iso20022)
```
[the remark this answers is not in the document]
Comment (Hdd87) [reply content/2.application.md]: fixed

File: content/2.application.md (hunk 1 above)
Comment (Hdd87) [reply content/2.application.md]: Fixed

File: content/2.application.md [hunk 3]
```diff
@@ -55,3 +55,14 @@ To simplify the development of a custom Payments CorDapp in the absence of the G
...
+### Supplementary parts of the Payments SDK
+
+1. SDK Payments:
+    - [Payments-Contracts](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-contracts)
+    - [Payments-Workflows](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-workflows)
+2. Payments App Sample:
+    - [Payments App Backend - Gevamu](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-app-sample/payments-app-backend)
+    - [Payments App Frontend - Client App](https://github.com/gevamu/corda-payments-sdk/tree/master/payments-app-sample/payments-app-frontend)
+3. [Java Classes created from ISO20022 XML schema](https://github.com/gevamu/corda-payments-sdk/tree/master/iso20022)
+    - [ISO20022 website](https://www.iso20022.org/)
```
Comment (d0rich) [inline content/2.application.md:68]: Delete Java classes reference. It is not needed.

```suggestion
3. [ISO20022 website](https://www.iso20022.org/)
```

File: content/2.application.md [hunk 4]
```diff
@@ -55,3 +55,14 @@ To simplify the development of a custom Payments CorDapp in the absence of the G
 
 ### PSP payments gateway
 The PSP payments gateway is an endpoint of a proprietary off-chain payments solution on the PSP side.
+
+### Supplementary parts of the Payments SDK
+
+1. SDK Payments:
```
Comment (d0rich) [inline content/2.application.md:61]: "Software Development Kit Payments" is not linguistically correct.

```suggestion
1. Payments SDK:
```

**Review by d0rich — CHANGES_REQUESTED** [review]

**Review by d0rich — APPROVED** [review]

**Pull request merged**